### DISCUSSION TITLE
react-select controlled

### DISCUSSION THREAD

**Discussion by andrconstruction** [react-hook-form/react-hook-form#1992] [Q&A] [answered]
Why should I use for a defaultValue object with a value and lablel, can I use only value to set default option?

**Comment by bluebill1049** [comment, marked as answer]
because that's react-select expected data shape.

**Reply by andrconstruction** [reply]
but react-select has a value option, how can I use this for a set value?

**Reply by andrconstruction** [reply]
Documentation 
![image](https://user-images.githubusercontent.com/7435330/86084942-a8f22380-baa6-11ea-9336-82f5d6ab1318.png)

but defaultValue on this method not working

**Reply by andrconstruction** [reply]
React-select value not setted after reset

**Reply by bluebill1049** [reply]
There is an example in the doc: https://codesandbox.io/s/react-hook-form-controller-079xx
https://react-hook-form.com/api#Controller
